### DISCUSSION TITLE
L' IP di Home Assistant

### DISCUSSION THREAD

**Discussion by Dregi56** [indomus/forum#1980] [HUB: Home Assistant]
Ho cambiato provider ed ho cambiato router. Prima l' IP di HA e Raspberry coincidevano. Il nuovo router Fritzbox ha assegnato ad HA un nuovo IP diverso da quello del raspberry. C'è un motivo? Posso "forzare" HA ad un IP uguale a Raspberry?

**Comment by snakuzzo** [comment]
Prenotazione indirizzi in base a MAC address del client...ogni router ha questa funzionalità.

**Reply by Dregi56** [reply]
E questo, stante l'attuale configurazione, mi può creare qualche problema?

**Reply by snakuzzo** [reply]
Non ricordo su HassOS, ma è di gran lunga la cosa peggiore.
Conviene fare in modo che ci sia assegnazione automatica direttamente dal router.

**Reply by Dregi56** [reply]
Quresto sembrerebbe anche a me, ma poi se ad un riavvio il router dovesse dare un IP diverso, che ne sarà dei reindirizzamenti delle porte?

**Reply by snakuzzo** [reply]
Non succedere mai... è un'impostazione memorizzata sul router!!!!
Comunque fai come meglio credi...
Su HassOS se vuoi impostare ip statico lo fai da settings > system > network

**Reply by snakuzzo** [reply]
Secondo te se questa impostazione fosse così volatile come dici, che senso avrebbe metterla?

**Reply by Dregi56** [reply]
Già. Tutti i discorsi che è meglio che ciascun dispositivo abbia un IP fisso? Perché Shelly dà la possibilità di determinarlo? Non sono domande polemiche, vorrei capire...

**Reply by snakuzzo** [reply]
Sono cazzate con la "C" maiuscola
